src/functions.php

```diff
@@ -597,11 +597,18 @@ function warning(string $message): void
     }
 }
 
+/**
+ * Writes a message to the output.
+ */
+function write(string|iterable $message, int $options = 0): void
+{
+    output()->write(parse($message), false, $options);
+}
+
 /**
  * Writes a message to the output and adds a newline at the end.
- * @param string|array $message
  */
-function writeln($message, int $options = 0): void
+function writeln(string|iterable $message, int $options = 0): void
```

Comment:
Iterable is wrong. Parse don’t take irerable.

Reply:
[Iterable is correct](https://github.com/symfony/symfony/blob/b5d4b33d5e65d956194c170fda5f8f40bc6a921a/src/Symfony/Component/Console/Output/OutputInterface.php#L36).